Packs/VMRay/Integrations/VMRay/VMRay.py

```diff
@@ -1342,6 +1363,38 @@ def get_summary_command():
     return_results(file_entry)
 
 
+def get_screenshots_command():
+    analysis_id = demisto.args().get('analysis_id')
+    check_id(analysis_id)
+
+    screenshots_data = get_screenshots(analysis_id)
+
+    file_results = []
+    screenshot_counter = 0
+    try:
+        with ZipFile(io.BytesIO(screenshots_data), 'r') as screenshots_zip:
+            index_log_data = screenshots_zip.read('screenshots/index.log', pwd=DEFAULT_ARCHIVE_PASSWORD)
```

Comment:
Will the password always be the default?
Can't the user change it, and then have to add a password parameter?